create-snowpack-app/app-template-11ty/snowpack.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  install: [
+    /* ... */
+  ],
+  plugins: [
+    ['@snowpack/plugin-run-script', { cmd: 'eleventy', watch: '$1 --watch' }],
+  ],
+  installOptions: {
+    /* ... */
+  },
+  devOptions: {
+    hmrDelay: 300,
+  },
+  buildOptions: {
+    /* ... */
+  },
+  proxy: {
+    /* ... */
+  },
+  mount: {
```

Comment:
for this template and all: lets move "mount" up to the top of the list, since it's defined in all templates. Same for "plugins" moving to the second position from the top.


```
module.exports = {
  mount: { ...
  plugins: [ ...
  // the rest of them
}
```